client/state/selectors/should-rate-limit-atomic-cache-clear.js

```diff
@@ -0,0 +1,14 @@
+import 'calypso/state/hosting/init';
+
+const FIVE_MINUTES_IN_MILLISECONDS = 300 * 1000;
```

Comment:
I think a minute is fine. Five minutes simply adds unnecessary delay to the user's workflow.